Doc/library/stdtypes.rst

```diff
@@ -1376,6 +1376,108 @@ objects that compare equal might have different :attr:`~range.start`,
    single: str (built-in class); (see also string)
    object: string
 
+.. _text-methods-summary:
+
+Text and Binary Sequence Type Methods Summary
+=============================================
+The following table summarizes the text and binary sequence types methods by
+category.
+
+
++--------------------------+------------------------------+------------------------------+
+| Category                 |  str methods                 |  bytes methods               |
++==========================+==============================+==============================+
+| Formatting               |  :meth:`str.format`          |                              |
++                          +------------------------------+------------------------------+
+|                          |  :meth:`str.format_map`      |                              |
++--------------------------+------------------------------+------------------------------+
+| Searching and Replacing  |  :meth:`str.find`            |  :meth:`bytes.find`          |
++                          +------------------------------+------------------------------+
+|                          |  :meth:`str.rfind`           |  :meth:`bytes.rfind`         |
++                          +------------------------------+------------------------------+
+|                          |  :meth:`str.index`           |  :meth:`bytes.index`         |
++                          +------------------------------+------------------------------+
+|                          |  :meth:`str.rindex`          |  :meth:`bytes.rindex`        |
```

Comment:
Perhaps the methods and their corresponding rmethods could be combined on the same line to make the table more compact.  If however the end result isn't more readable, you could leave them separate.

Reply:
I added the methods and rmethods on the same lines, but I did it 2 different ways so that you could pick which one you prefer.  `find\rfind` and `index\rindex` are done as columns while `split\rsplit`, `ljust\rjust`, and `lstrip/rstrip` are done as `or`.